enum.c

```diff
@@ -3970,7 +3970,7 @@ int_range_sum(VALUE beg, VALUE end, int excl, VALUE init)
  *   { 1 => 10, 2 => 20 }.sum {|k, v| k * v }  #=> 50
  *   (1..10).sum                               #=> 55
  *   (1..10).sum {|v| v * 2 }                  #=> 110
- *   [Object.new].each.sum                     #=> TypeError
+ *   [Object.new].sum                          #=> TypeError
```

Comment:
> Actually, this example is not proper in here since it's Array#sum, not Enumerable#sum.

Can you come up with something which is really `Enumerable#sum`?

Reply:
@k0kubun I came up with range of string, which calls `Enumerable#sum` and causes `TypeError` as intended.

Reply:
cool 🙂